rubyau/constitution.md

```diff
@@ -1,468 +1,779 @@
+  1. this power of delegation; or
+  2. a duty imposed on the Committee by the Act or any other law.
+
+2. The delegation must be in writing and may be subject to the conditions and limitations the Committee considers appropriate.
+
+3. The Committee may, in writing, revoke a delegation wholly or in part.
+
+### Division 2—Composition of Committee and duties of members
+
+#### 44 Composition of Committee
+
+1. The Committee consists of—
+  1. a President; and
+  2. a Vice-President; and
+  3. a Secretary; and
+  4. a Treasurer; and
+  5. four ordinary members (if any) elected under rule 53.
+
+#### 45 General Duties
+
+1. As soon as practicable after being elected or appointed to the Committee, each committee member must become familiar with these Rules and the Act.
+
+2. The Committee is collectively responsible for ensuring that the Association complies with the Act and that individual members of the Committee comply with these Rules.
+
+3. Committee members must exercise their powers and discharge their duties with reasonable care and diligence.
+
+4. The Term of a committee member is 12 months and the term of the Treasurer is 24 months.
+
+5. Committee members must exercise their powers and discharge their duties—
+  1. in good faith in the best interests of the Association; and
+  2. for a proper purpose.
+
+6. Committee members and former committee members must not make improper use of—
+  1. their position; or
+  2. information acquired by virtue of holding their position—
+
+  so as to gain an advantage for themselves or any other person or to cause detriment to the Association.
+
+7. In addition to any duties imposed by these Rules, a committee member must perform any other duties imposed from time to time by resolution at a general meeting.
+          
+#### 46 President and Vice-President
+
+1. Subject to subrule 2, the President or, in the President's absence, the Vice-President is the Chairperson for any general meetings and for any committee meetings.
+
+2. If the President and the Vice-President are both absent, or are unable or unwilling to preside, the Chairperson of the meeting can be any person that the President or Vice-President delegates to the role or if they are unable to do so, then any person voted by the majority of the Committee.
+
+#### 47 Secretary
+
+1. The Secretary must perform any duty or function required under the Act to be performed by the secretary of an incorporated association.
+
+  ##### Example
+  Under the Act, the secretary of an incorporated association is responsible for lodging documents of the association with the Registrar.
+
+2. The Secretary must—
+  1. maintain the register of members in accordance with rule 18; and
+  2. keep custody of the common seal (if any) of the Association and, except for the financial records referred to in rule 70, subrule 3, all books, documents and securities of the Association in accordance with rules 72 and 75; and
```

Comment:
Rule 70 subrule 3 does not appear to exist. Rules 72 and 75 do not appear relevant. I’m assuming these rule numbers are obsolete.

Someone needs to figure out what rules are intended to be referenced here and fix these references.